indico/modules/users/templates/dashboard.html

```diff
@@ -49,6 +49,17 @@ <h3>{{ user.full_name }}</h3>
                                 </div>
                             {% endif %}
                             {{ labels(user) }}
+                            <br>
+                            <div class="ui small label">
+                                {% trans %}Last login date{% endtrans %}
+                                <div class="detail">
```

Comment:
the content of this div should be indented

Reply:
Did you mean for `<div class="detail">`?

Reply:
yes, anything that's in a nested element (regardless if html or jinja) gets a new level of indentation

Reply:
I've intended the if conditions in div element, is it correct?

Reply:
yep